### PR TITLE
[15.0][MIG] rma_purchase_analytic

### PR DESCRIPTION
**Pull request by JasminSForgeFlow**

### CODE REVIEW

File: rma_purchase_analytic/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# © 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline rma_purchase_analytic/__manifest__.py]: Can you relicense to ForgeFlow?

File: rma_purchase_analytic/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+{
+    "name": "Analytic Account in RMA purchase",
+    "version": "15.0.1.0.0",
+    "author": "ForgeFlow," "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "website": "https://github.com/ForgeFlow/stock-rma",
+    "category": "Analytic",
+    "depends": ["rma_account", "rma_analytic", "purchase_analytic"],
+    "data": [],
```
Comment (AaronHForgeFlow) [inline rma_purchase_analytic/__manifest__.py]: This can be deleted

File: rma_purchase_analytic/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
```
Comment (AaronHForgeFlow) [inline rma_purchase_analytic/models/__init__.py:1]: This can be deleted

File: rma_purchase_analytic/wizards/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2023 ForgeFlow S.L.
```
Comment (AaronHForgeFlow) [inline rma_purchase_analytic/wizards/__init__.py]: This can be deleted

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM! Minor comments

File: rma_purchase_analytic/models/__init__.py (hunk 3 above)
Comment (JasminSForgeFlow) [reply rma_purchase_analytic/models/__init__.py:1]: is it required to be removed? its licence line

Comment (AaronHForgeFlow) [reply rma_purchase_analytic/models/__init__.py:1]: In the init file is not necessary, it is just the imports for the files. It does not contain actual code. It is not big deal, but you can change it, just to "standarize" with OCA

File: rma_sale_operating_unit/README.rst [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Contributors
+------------
+
+* Aaron Henriquez <ahenriquez@eficent.com>
+
+
+Maintainer
+----------
+
+This module is maintained by Eficent.
```
Comment (AaronHForgeFlow) [inline rma_sale_operating_unit/README.rst]:
```suggestion
This module is maintained by ForgeFlow.
```

File: rma_sale_operating_unit/README.rst [hunk 6]
```diff
@@ -0,0 +1,28 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+=============================
+RMA Sale with Operating Units
```
Comment (AaronHForgeFlow) [inline rma_sale_operating_unit/README.rst]:
```suggestion
RMA Purchase with Operating Units
```

File: rma_sale_operating_unit/README.rst [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+=============================
+
+This module introduces the following features:
+
+* Adds the operating unit to the quotation
+
+Usage
```
Comment (AaronHForgeFlow) [inline rma_sale_operating_unit/README.rst]: Delete this. It is empty.

**Review by AaronHForgeFlow — COMMENTED** [review]
Thanks @JasminSForgeFlow  some minor comments before the merge

**Comment by JasminSForgeFlow** [comment]
@AaronHForgeFlow it is now rebased, was issue in rebase

**Comment by AaronHForgeFlow** [comment]
@JasminSForgeFlow , there are some merge commits in the commit history that should not be there. Also there are still conflicts in the PR so I can still not merge. Have you fetched the latest code in 15.0 before the rebase?

**Comment by JasminSForgeFlow** [comment]
> @JasminSForgeFlow , there are some merge commits in the commit history that should not be there. Also there are still conflicts in the PR so I can still not merge. Have you fetched the latest code in 15.0 before the rebase?

I did rebase but maybe something went wrong. I have just followed the migration process and pushed it so it is up to date now

**Comment by AaronHForgeFlow** [comment]
@JasminSForgeFlow now the history looks good. Thanks.

**Review by AaronHForgeFlow — APPROVED** [review]
It works same as in previous versions.

I miss some features like filling in the analytic account in the RMA line when selecting a source PO, but that can be done in another PR.

**Pull request merged**